core/compilers.ts

```diff
@@ -33,8 +33,7 @@ export function compile(code: string, path: string): string {
   }
   if (path.endsWith(".ipynb")) {
     const notebookAsJson = stripNotebookOutputs(JSON.parse(code), path);
-    // TODO(ekrekr): base64 encode the notebook as a string instead.
-    return `exports.asBase64String = () => \`${JSON.stringify(notebookAsJson)}\``;
+    return `exports.asString = \`${JSON.stringify(notebookAsJson)}\``;
```

Comment:
IMO this should be returned as JSON, I think that would be a lot cleaner. But I won't block.

Reply:
Fair point, it's not really a string! Done.